$OpenBSD: patch-config_lib,v 1.7 2020/04/07 15:13:34 solene Exp $
Don't build with -O1. The reason they add it is irrevelant to OpenBSD anyway.
Index: config.lib
--- config.lib.orig
+++ config.lib
@@ -1525,7 +1525,7 @@ make_cflags_and_ldflags() {
 		cc_build_is_gcc=`basename "$cc_build" | grep "gcc" 2>/dev/null`
 		if [ -n "$cc_build_is_gcc" ]; then
 			# Just add -O1 to the tools needed for building.
-			CFLAGS_BUILD="$CFLAGS_BUILD -D_FORTIFY_SOURCE=2 -O1"
+			CFLAGS_BUILD="$CFLAGS_BUILD -D_FORTIFY_SOURCE=2"
 			if [ "$os" = "MINGW" ]; then
 				# Prevent undefined references when _FORTIFY_SOURCE > 0
 				LDFLAGS_BUILD="$LDFLAGS_BUILD -fstack-protector"
